access/email/config.go

```diff
@@ -183,10 +189,29 @@ func (c *Config) CheckAndSetDefaults() error {
 		c.Log.Severity = "info"
 	}
 
-	// Validate emails in user aliases
-	for _, e := range c.Delivery.Recipients {
-		if !lib.IsEmail(e) {
-			return trace.BadParameter("Invalid email address %v in users.recipients", e)
+	if len(c.Delivery.Recipients) > 0 {
+		if len(c.RoleToRecipients) > 0 {
+			return trace.BadParameter("provide either delivery.recipients or role_to_recipients, not both.")
+		}
+
+		c.RoleToRecipients = config.RecipientsMap{
+			types.Wildcard: c.Delivery.Recipients,
+		}
+		c.Delivery.Recipients = nil
+	}
+
+	if len(c.RoleToRecipients) == 0 {
+		return trace.BadParameter("missing required value role_to_recipients.")
+	}
+	if len(c.RoleToRecipients[types.Wildcard]) == 0 {
+		return trace.BadParameter("missing required value role_to_recipients[%v].", types.Wildcard)
+	}
+
+	for role, recipientsList := range c.RoleToRecipients {
+		for _, recipient := range recipientsList {
+			if !lib.IsEmail(recipient) {
+				return trace.BadParameter("Invalid email address %v in role_to_recipients.%s", recipient, role)
```

Comment:
Nit: Don't capitalize returned error messages.
```suggestion
				return trace.BadParameter("invalid email address %v in role_to_recipients.%s", recipient, role)
```